gateway/src/node/mod.rs

```diff
@@ -636,6 +660,13 @@ impl<St> Gateway<St> {
             shutdown.fork("mixnet_handling::Listener"),
         );
 
+        let shared_session_stats = self.session_stats.take().unwrap_or_default();
+        self.start_stats_collector(
+            active_clients_store.clone(),
+            shared_session_stats,
+            shutdown.fork("statistics::sessionCollector"),
```

Comment:
`camelCase` makes me think of Java and immediately get chills down my spine. Let's use `PascalCase` ;)
Also, isn't it called `StatisticsCollector` or `GatewayStatisticsCollector`?

Reply:
Sorry for the chills :)

Fixed in 65b83627fc31506ef8725adf0cd21632f68a55aa